jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -135,6 +138,22 @@ class Terminal extends React.Component {
     this.attachResizeListener();
   }
 
+  /**
+    Write a welcome message to the given term instance.
+    @param {Object} term The xterm instance.
+    @param {String} message The optional response message from the server.
+  */
+  _welcome(term, message) {
```

Comment:
`_writeWelcomeMessage` ?

Reply:
AFAIK welcome can be used as a transitive verb, so I'd say I like _welcome more, but maybe I am just being more influenced by Go than Java...